docs/pages/config.mdx

```diff
@@ -90,19 +99,8 @@ The global configuration keys are all optional.
   The keys are the names of the enumerations.
   The values are arrays of the strings for the values the enumeration can take.
 
-- **`tables`**: a record of tables. The keys in the record are table names.
```

Comment:
why was this removed?

Reply:
Because in the current version we have two separate `tables` lines. [See here for the other one](https://github.com/latticexyz/mud/blob/main/docs/pages/config.mdx?plain=1#L61)